docker/worldping-api/worldping-api.ini

```diff
@@ -20,7 +20,7 @@ cert_key =
 admin_key = changeme
 
 [raintank]
-graphite_url = http://graphite-api:8888/
+graphite_url = http://metrictank:6063/
```

Comment:
legacy.  older versions of the worldping-app still point to worldping-api.raintank.io/api/graphite for querying the TSDB.